frontend/app/openproject-app.js

```diff
@@ -44,9 +44,9 @@ require('angular-animate');
 require('angular-aria');
 require('angular-modal');
 
-if (I18n.locale === 'de') {
-  require('angular-i18n/angular-locale_de-de');
-}
+// depends on the html element having a 'lang' attribute
+var documentLang = angular.element('html').attr('lang');
+require('angular-i18n/angular-locale_' + documentLang + '.js');
```

Comment:
You should provide a fallback here since (at least) running karma that attribute does not exist, and thus fails requiring `  './angular-locale_undefined.js'`.